exercises/raindrops/src/test/java/RaindropConverterTest.java

```diff
@@ -17,28 +17,26 @@ public void soundFor1Is1() {
 
     @Ignore("Remove to run test")
     @Test
-    public void soundFor3OrNumberWithOnlyDistinctFactorOf3() {
+    public void soundFor3IsPling() {
         assertEquals("Pling", raindropConverter.convert(3));
-        assertEquals("Pling", raindropConverter.convert(6));
-        assertEquals("Pling", raindropConverter.convert(9));
-        assertEquals("Pling", raindropConverter.convert(27));
     }
 
     @Ignore("Remove to run test")
     @Test
-    public void soundFor5OrNumberWithOnlyDistinctFactorOf5IsPlang() {
+    public void soundFor5IsPlang() {
         assertEquals("Plang", raindropConverter.convert(5));
-        assertEquals("Plang", raindropConverter.convert(10));
-        assertEquals("Plang", raindropConverter.convert(25));
-        assertEquals("Plang", raindropConverter.convert(3125));
     }
 
     @Ignore("Remove to run test")
     @Test
-    public void soundFor7OrNumberWithOnlyDistinctFactorOf7IsPlong() {
+    public void soundFor7IsPlong() {
         assertEquals("Plong", raindropConverter.convert(7));
-        assertEquals("Plong", raindropConverter.convert(14));
-        assertEquals("Plong", raindropConverter.convert(49));
+    }
+
+    @Ignore("Remove to run test")
+    @Test
+    public void soundForNumber6WithOnlyDistinctFactorOf3IsPling() {
```

Comment:
The test names are a bit inconsistent. Some of them just state `the sound for this number is...` whereas others state `the sound for this number with factor... is...`. I think either is fine but it would be good to be consistent 🙂 Also some of them are of the form `soundForX` whereas others are `soundForNumberX`. Again I think either is fine but it would be good to be consistent 🙂 

If you want to stick to `the sound for this number with factor... is...` then I find the phrasing `with only distinct factor` a bit confusing. What exactly do you mean by that? The numbers do sometimes, like in this case, have other factors they're just not in the list of numbers that translate to raindrop sounds. I would maybe prefer something like `soundForXIsYAsItHasFactorZ` or something. What do you think? 🙂